CoordgenFragmentBuilder.cpp

```diff
@@ -1045,6 +1045,12 @@ void CoordgenFragmentBuilder::simplifyRingSystem(
                         /* don't separate rings of bicyclo (1,1,1) pentane so we can use a template instead */
                         n++;
                     }
+                    if (r->fusionAtoms.at(ringCounter).size() == 3 &&
+                        r->size() == 5 &&
+                        fusedRing->size() == 5) {
```

Comment:
What's the cost of this?

Like, if we don't separate the rings, but it isn't actually norborane... Would that make the generated structure worse? Or would that break matching to some other templates?

Reply:
I'm not sure there's another way to fuse two pentagons on 3 consecutive vertices other than the structure of norbornane... but if we don't separate here and don't find a template then the structure will be generated with the "as planar as possible" approach, which should still do a pretty good job here